experimenter/experimenter/nimbus_ui_new/static/.eslintrc.json

```diff
@@ -0,0 +1,20 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:prettier/recommended"
+    ],
+    "parserOptions": {
+        "ecmaVersion": 12,
+        "sourceType": "module"
+    },
+    "rules": {
+        "prettier/prettier": "error"
+    },
+    "globals": {
+        "$": "readonly"
+    }
+}
```

Comment:
Missing newline

Reply:
I don't usually setup linters/formatters for config files?